ql/src/test/results/clientpositive/llap/uniontype_cbo.q.out

```diff
@@ -0,0 +1,171 @@
+PREHOOK: query: CREATE TABLE utable (cu UNIONTYPE<INTEGER, STRING>)
```

Comment:
I double-checked the diff between the result of the almost latest master and this file would be expected.

```
16c16
< PREHOOK: query: EXPLAIN CBO
---
> PREHOOK: query: EXPLAIN CBO 
23c23
< POSTHOOK: query: EXPLAIN CBO
---
> POSTHOOK: query: EXPLAIN CBO 
29a30,33
> CBO PLAN:
> HiveTableFunctionScan(invocation=[inline(ARRAY(ROW(create_union(0, 10, _UTF-16LE'ten':VARCHAR(2147483647) CHARACTER SET "UTF-16LE")), ROW(create_union(1, 10, _UTF-16LE'ten':VARCHAR(2147483647) CHARACTER SET "UTF-16LE"))))], rowType=[RecordType(RecordType(INTEGER $tag_0, VARCHAR(2147483647) $tag_1) col1)])
>   HiveTableScan(table=[[_dummy_database, _dummy_table]], table:alias=[_dummy_table])
> 
52a57,60
> CBO PLAN:
> HiveProject(cu=[$0])
>   HiveTableScan(table=[[default, utable]], table:alias=[utable])
> 
72a81,84
> CBO PLAN:
> HiveProject(_o__c0=[extract_union($0)])
>   HiveTableScan(table=[[default, utable]], table:alias=[utable])
> 
113a126,129
> CBO PLAN:
> HiveProject(u_age_name=[create_union(MOD($0, 2), $2, $1)])
>   HiveTableScan(table=[[default, author]], table:alias=[author])
> 
141a158,161
> CBO PLAN:
> HiveProject(u_age_name=[$0])
>   HiveTableScan(table=[[default, uauthor]], table:alias=[uauthor])
> 
```